### ISSUE TITLE
pgbackrest fails to proceed without log file despite the notice

### ISSUE THREAD

**Issue by vthriller** [pgbackrest/pgbackrest#619]
This is what I see using pgBackRest 2.06 on Ubuntu 16.04.5 via pgdg repo:

```
# sudo -u pgbackrest pgbackrest stanza-create --stanza=xxx
WARN: unable to open log file '/var/log/pgbackrest/xxx-stanza-create.log': Permission denied
      NOTE: process will continue without log file.
ERROR [041]: : unable to open log file '/var/log/pgbackrest/xxx-stanza-create.log': Permission denied
ERROR: [124]: process terminated due to an unhandled exception:
       syswrite() on closed filehandle $hLogFile at /loader/0x1278108/pgBackRest/Common/Log.pm line 759.
        at /loader/0x1278108/pgBackRest/Main.pm line 12.
        pgBackRest::Main::__ANON__("syswrite() on closed filehandle \$hLogFile at /loader/0x127810"...) called at /loader/0x1278108/pgBackRest/Common/Log.pm line 759
        pgBackRest::Common::Log::log("ERROR", "unable to open log file '/var/log/pgbackrest/xxx-stanza-creat"..., 41) called at /loader/0x1278108/pgBackRest/Common/Log.pm line 600
        pgBackRest::Common::Log::logErrorResult(41, "unable to open log file '/var/log/pgbackrest/xxx-stanza-creat"..., "") called at /loader/0x1278108/pgBackRest/Common/Log.pm line 138
        pgBackRest::Common::Log::logFileSet(pgBackRest::Storage::Local=HASH(0x1ccdf88), "/var/log/pgbackrest/xxx-stanza-create") called at /loader/0x1278108/pgBackRest/Main.pm line 200
        eval {...} called at /loader/0x1278108/pgBackRest/Main.pm line 286
        pgBackRest::Main::main("stanza-create") called at (eval 267) line 1
        eval '($iResult, $bErrorC, $strMessage) = pgBackRest::Main::main(\'stanza-create\')' called at (eval 267) line 0
```

Sure I fixed this for myself with a single `chown`, but still…

**Comment by dwsteele** [comment]
Yeah, that doesn't look right.  It seems the C code is more tolerant of this condition than the Perl code.  We'll have a look at it.

**Comment by dwsteele** [comment]
Fix committed at 03b9db9aa2e8.  Let me know if you'd like to use a different name in the release notes.
